### DISCUSSION TITLE
Showing temperature and humidity values ?

### DISCUSSION THREAD

**Discussion by GooSe1977** [kalkih/mini-graph-card#949] [General]
I'm using this code

```type: custom:mini-graph-card
entities:
  - sensor.balkon_temperature
name: Balkon
hours_to_show: 24
show:
  name: true
  legend: false
  icon: false
  labels: false
color_thresholds:
  - value: -5
    color: '#98F5FF'
  - value: 0
    color: '#1C86EE'
  - value: 5
    color: '#B4EEB4'
  - value: 10
    color: '#90EE90'
  - value: 15
    color: '#54FF9F'
  - value: 20
    color: '#f39c12'
  - value: 21
    color: '#d35400'
  - value: 21.5
    color: '#c0392b'

```

and would like to show the humidity in top right section als a value, ist this possible ?

**Comment by ildar170975** [comment]
1. Do not post unformatted code. Currently your code cannot be analysed.
2. Check this - https://github.com/kalkih/mini-graph-card#showing-additional-info-on-the-card

**Reply by GooSe1977** [reply]
Sorry for the probably dumb question - how to I upload the code correctly ?
I tried the description in your link, but I only got the humidity to show as a graph, not a a second number.

**Reply by ildar170975** [reply]
Use triple "`":
```
some code
```

![image](https://user-images.githubusercontent.com/71872483/230719355-6c248486-74d8-4010-a8e3-acabe7784542.png)

**Reply by ildar170975** [reply]
> Sorry for the probably dumb question - how to I upload the code correctly ? I tried the description in your link, but I only got the humidity to show as a graph, not a a second number.

On the link there are 3 entities, for 2 of them states are not displayed (`show_state: false`), graphs are displayed; for 1 entity only state is displayed.

**Comment by GooSe1977** [comment]
OK, I got this running now ;-) THX !
